echo/frontend/src/components/participant/ParticipantOnboardingCards.tsx

```
@@ -175,115 +179,130 @@ const ParticipantOnboardingCards = ({
+          <div className="mt-4 flex items-center justify-between">
+            <div className="flex space-x-2">
+              {allSlides.map((_, index) => (
+                <div
+                  key={index}
+                  className={`h-2 w-2 rounded-full transition-all duration-200 ${
+                    index === currentSlideIndex
+                      ? "w-4 bg-blue-500"
+                      : "bg-gray-300"
+                  }`}
+                ></div>
+              ))}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)